index.d.ts

```diff
@@ -7,6 +7,13 @@ export type ErrorObject = {
 	code?: string;
 } & JsonObject;
 
+export type ErrorLike = {
+	name: string;
+	stack: string;
+	message: string;
+	code?: string;
+} & JsonObject;
```

Comment:
`& JsonObject` is not correct here because the JS code is not actually ensuring that.

Reply:
Should we also add `cause` as optional? Same with `ErrorObject`.

Reply:
> & JsonObject is not correct here because the JS code is not actually ensuring that.

I suppose the solution is to use a generic Record then

>Should we also add cause as optional? Same with ErrorObject.

Done